app/scripts/lib/assertion.js

```diff
@@ -24,14 +24,13 @@ function (P, jwcrypto) {
     // browser has native crypto, no need to fetch entropy from the server.
     try {
       if (window.crypto && window.crypto.getRandomValues) {
-        return P(true); /*jshint ignore:line*/
+        return P(true);
       }
     } catch(e) {
       // some browsers blow up when trying to query window.crypto.
     }
 
     // wah wah, we need to get entropy from the server.
-    //jshint validthis: true
```

Comment:
As a note, this has been added in `v1.0.0` of ESLint as [no-invalid-this](http://eslint.org/docs/1.0.0/rules/no-invalid-this)